tests/NTest_file_lfs.lua

```diff
@@ -0,0 +1,103 @@
+-- tests for file_lfs module
+-- resource.lua with index.html or favicon.ico or test.txt file needs to be embedded in LFS
+-- the file should be larger than 1kB
+
+-- luacheck: globals file
+-- luacheck: new read globals node.LFS.resource
+file = require("file_lfs")
+
+-- check standard SPIFFS file functions
+require("NTest_file")
+
+local N = require('NTest')("file_lfs")
+
+N.test('resource.lua in LFS', function()
+  ok(node.LFS.resource~=nil, "resource.lua embedded in LFS")
+end)
```

Comment:
Please change the code to the following:

```Lua
-- luacheck: globals file
-- luacheck: new read globals node.LFS.resource
file = require("file_lfs")

local Nt = ...
Nt = (Nt or require "NTest")

-- check standard SPIFFS file functions
loadfile("NTest_file.lua")(Nt)

local N = Nt("file_lfs")

N.test('resource.lua in LFS', function()
  ok(node.LFS.resource~=nil, "resource.lua embedded in LFS")
end)
```

This is for two reasons:
1. we changed the tests to allow passing a preconfigured NTest instnce which must be used.
2. Please don't require tests, just dofile them.

I stumbled across this because the original version didn't run for me.

Reply:
It's interesting. This version did run for me. Anyway it's adjusted as requested.